acme.sh

```diff
@@ -520,13 +520,13 @@ _get() {
   if _exists "curl" ; then
     if [ "$onlyheader" ] ; then
       $CURL -I -A "User-Agent: $USER_AGENT" $url
-      if [[ $? -gt 0 ]] ;then
-       _err "Curl failed with exit code $?, trying running script in DEBUG=2 mode"
+      if [ $? -gt 0 ] ;then
```

Comment:
wget can return error codes from 0-8.
curl can return different error levels from 1 to 90

I can flip the logic to succeed when == 0, but then the final 'good' exit point will no longer be at the bottom, which seemed to be a convention that was being followed? (maybe not?)